client/state/initial-state.js

```diff
@@ -150,7 +150,7 @@ export default function createReduxStoreFromPersistedInitialState( reduxStoreRea
 		if ( shouldAddSympathy() ) {
 			// eslint-disable-next-line no-console
 			console.log(
-				'%cSkipping initial state rehydration to recreate first-load experience.',
+				'%cSkipping initial state rehydration. (This runs during random page requests in the Calypso development environment, to simulate a user loading the application for the first time.)',
```

Comment:
seems good even if a bit verbose. one note is that it's not just about "the first time" but any first load when the cache is empty. for many customers this is a quite-frequent event.

Reply:
Thanks!  I'll plan to get this out on Monday.

And good point... maybe I'll change it to something more like "simulate loading the application with an empty cache" before merging.